release/src/module_dependencies/main.py

```diff
@@ -197,4 +200,61 @@ def getImmediateDependents(moduleNameList, moduleDetailsJSON):
                     
     return moduleDetailsJSON
 
+# Updates the stdlib dashboard in README.md
+def updateStdlibDashboard(moduleDetailsJSON):
+    try:
+        readMeFile = urlOpenWithRetry("https://raw.githubusercontent.com/ballerina-platform/ballerina-standard-library/main/README.md")
+    except:
+        print('Failed to read README.md file')
+        sys.exit()
+
+    updatedReadMeFile = ''
+
+    for line in readMeFile:
+        processedLine = line.decode("utf-8")
+        updatedReadMeFile += processedLine
+        if "|:---:|:---:|:---:|:---:|:---:|:---:|:---:|" in processedLine:
+            break
+
+    # Modules in levels 0 and 1 are categorized under level 1
+    # A single row in the table is created for each module in the module list    
+    levelColumn = 1
+    currentLevel = 0
+    for module in moduleDetailsJSON['modules']:
+        if module['level'] > currentLevel:
+            levelColumn = module['level']
+            currentLevel = module['level']
+
+        row = ("|" + str(levelColumn) + "|" + 
+        "[" + module['name'].split('-')[-1] + "](" + BALLERINA_ORG_URL + module['name'] + ")| " + 
+
+        "[![GitHub Release](" + GITHUB_BADGE_URL + "release/ballerina-platform/" + module['name'] + ".svg?label=)]" + 
```

Comment:
Can't we use a separate variable for `ballerina-platform`? I see it is used in many places.

Reply:
Done